src/form-engine.test.tsx

```diff
@@ -1047,6 +1098,86 @@ describe('Form engine component', () => {
     });
   });
 
+  describe('Diagnisis field', () => {
+    it('should test addition of a diagnosis', async () => {
+      await act(async () => {
+        renderForm(null, diagnosisForm);
+      });
+      const addButtons = screen.getAllByRole('button', { name: 'Add' });
+      expect(addButtons.length).toBeGreaterThan(0);
```

Comment:
We always have a single "Add" button in a repeat section.

Reply:
<img width="826" alt="Screenshot 2025-01-17 at 12 22 15" src="https://github.com/user-attachments/assets/f032bb9a-c7dd-4a2d-aebf-b31833558ba4" />

This is the form im using as my mock